docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+	{
+		"name": "PublicLibFunctionsDoNotReturnNestedArrays",
+		"summary": "Calls to public library functions (internal functions are safe) that return nested arrays return only zeroes.",
```

Comment:
Does it apply to fixed-sized and dynamically sized arrays? What is the minimum nesting depth?

Reply:
Could you clarify this, please? So it needs to be a dynamically sized array (including `string` and `bytes`) inside a dynamically sized array?

Reply:
Sorry, I got it confused with another issue. It doesn't have to be dynamically sized, the problem happens with fixed-size 2d arrays already.